zeebe/expression-language/src/main/java/io/camunda/zeebe/el/impl/FeelEvaluationResult.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.el.impl;
+
+import static io.camunda.zeebe.util.buffer.BufferUtil.cloneBuffer;
+import static scala.jdk.javaapi.CollectionConverters.asJava;
+
+import io.camunda.zeebe.el.EvaluationResult;
+import io.camunda.zeebe.el.EvaluationWarning;
+import io.camunda.zeebe.el.Expression;
+import io.camunda.zeebe.el.ResultType;
+import java.time.Duration;
+import java.time.Period;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.function.Function;
+import org.agrona.DirectBuffer;
+import org.camunda.feel.syntaxtree.Val;
+import org.camunda.feel.syntaxtree.ValBoolean;
+import org.camunda.feel.syntaxtree.ValContext;
+import org.camunda.feel.syntaxtree.ValDate;
+import org.camunda.feel.syntaxtree.ValDateTime;
+import org.camunda.feel.syntaxtree.ValDayTimeDuration;
+import org.camunda.feel.syntaxtree.ValList;
+import org.camunda.feel.syntaxtree.ValLocalDateTime;
+import org.camunda.feel.syntaxtree.ValNull$;
+import org.camunda.feel.syntaxtree.ValNumber;
+import org.camunda.feel.syntaxtree.ValString;
+import org.camunda.feel.syntaxtree.ValYearMonthDuration;
+
+final class FeelEvaluationResult implements EvaluationResult {
```

Comment:
❓ Should this class be `public` like the others in this package? I think the Scala version of this was a public class (since it wasn't defined as private and in Scala things are public-by-default). Plus, we return instances of this class from the `FeelExpressionLanguage` (but only as the specialized type of an interface at runtime).

Reply:
I'd lean towards no because this class is an implementation detail and not part of our API. Is there any reason why we _should_ expose it as public (other than that it was public before)?

Reply:
I was wondering about the same thing. I can't think of any specific reason right now to say it _should_ to be public (since it's not part of our API) 👍